frontend/src/components/profile/components/profile-info/profile-main/profile-main.tsx

```diff
@@ -0,0 +1,145 @@
+import AddSection from '../add-section/add-section';
+import CareerCard from '../career-card/career-card';
+import EditSection from '../edit-section/edit-section';
+import EducationCard from '../education-card/education-card';
+import Tag from '../tag/tag';
+import  './profile-main.scss';
+
+interface CareerJourney {
+  id: string;
+  title: string;
+  position: string;
+  company: string
+  startDate: Date;
+  endDate: Date;
+}
+
+// FROM DB
+const careerJourneyData: CareerJourney[] = [
+  {
+    id: '1',
+    title: 'Fullstack JS Developer',
+    position:'Fullstack JS Developer',
+    company:'Binary Studio',
+    startDate: new Date('2020-01-30T03:24:00'),
+    endDate: new Date(),
+  },
+  {
+    id: '2',
+    title: 'Fullstack JS Developer',
+    position:'Fullstack JS Developer',
+    company:'Binary Studio',
+    startDate: new Date('2021-12-17T03:24:00'),
+    endDate: new Date(),
+  },
+];
+
+interface Education {
```

Comment:
You should move it to profile-info/interface, and after that import this interface and extends it in props and other interfaces.

We should use DRY and don't duplicate our code.